app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,35 @@
 <resources>
     <string name="app_name">Forzenbook</string>
+    <!--
+    Create Account Page
+    -->
+    <string name="top_bar_text_create_account">Create Account</string>
+    <string name="error_create_account">There was an error when\n
```

Comment:
Why is there a line break in here? That won't scale to every screen. The text should just wrap where it wraps unless you can really justify this

Reply:
thats a good point, removed the break line.